packages/ui/BodyText/tests/BodyText-specs.js

```diff
@@ -50,4 +50,15 @@ describe('BodyText Specs', () => {
 		const actual = subject.hasClass(css.centered);
 		expect(actual).toBe(expected);
 	});
+
+	test('should support changing the component element', () => {
+		const componentTag = 'address';
+		const subject = shallow(
+			<BodyText component={componentTag} />
+		);
+
+		const expected = componentTag;
+		const actual = subject.getElement().type;
+		expect(actual).toBe(expected);
+	});
```

Comment:
Let's add one more test for supporting function components.

Reply:
Would a functional component being used in the prop be any different than a string?

Reply:
I went ahead and added a test, just for good measure.